kymatio/scattering1d/frontend/tensorflow_frontend.py

```diff
@@ -15,7 +15,27 @@ def __init__(self, J, shape, Q=1, T=None, max_order=2, oversampling=0,
         ScatteringBase1D.build(self)
         ScatteringBase1D.create_filters(self)
 
+
 ScatteringTensorFlow1D._document()
 
 
-__all__ = ['ScatteringTensorFlow1D']
+class TimeFrequencyScatteringTensorflow(
+        ScatteringTensorFlow, TimeFrequencyScatteringBase):
+    def __init__(self, *, J, J_fr, shape, Q, T=None, oversampling=0, Q_fr=1,
+            F=None, oversampling_fr=0, out_type='array', backend='tensorflow', name="TimeFrequencyScatteringScattering"):
+        ScatteringTensorFlow.__init__(self,name=name)
+        TimeFrequencyScatteringBase.__init__(self, J=J, J_fr=J_fr, shape=shape,
+            Q=Q, T=T, oversampling=oversampling, Q_fr=Q_fr, F=F,
+            oversampling_fr=oversampling_fr, out_type=out_type, backend=backend)
+        TimeFrequencyScatteringBase._instantiate_backend(
+            self, 'kymatio.scattering1d.backend.')
+        TimeFrequencyScatteringBase.build(self)
+        TimeFrequencyScatteringBase.create_filters(self)
+    
+
+
+
+__all__ = ['ScatteringTensorflow1D', 'TimeFrequencyScatteringTensorflow']
```

Comment:
capital F in TensorFlow SVP

weird that your tests don't fail??

Reply:
i have incorporated your feedback and pushed again. Indeed the tests didn't fail for some reasons..

Reply:
after cyrus's feedback, i think it's clear that your tests weren't testing the right thing